README.md

```diff
@@ -275,9 +275,9 @@ its UUID with the `balena device move` command:
 $ BALENARC_DATA_DIRECTORY=. balena device move 7654321
 ? Select an application (Use arrow keys)
 ❯ altApp (fincm3)
+  anotherApp (raspberry-pi) 
```

Comment:
Can we remove this one please, as it looks like one of your own apps and is confusing if the user's followed the instructions (there should only be one app!).

Reply:
I added it to illustrate that when moving applications, other applications might be listed too especially for people following the tutorial if they already have several apps. 
Won't it be more confusing if something similar appeared? Perhaps we should clarify that line in the description below it?

Reply:
OK, I can see that. :)